modeling/model_utils/constants.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+import git
+from sklearn.model_selection import StratifiedKFold
+
+repo = git.Repo(".", search_parent_directories=True)
+ROOT_DIR_PATH = repo.working_tree_dir
+
+LOG_CONFIG_PATH = os.path.join(ROOT_DIR_PATH, "conf/logging.conf")
+
+# mlrun path
+TRACKING_URI_PATH = "file:" + os.path.join(ROOT_DIR_PATH, "mlruns")
+ARTIFACT_PATH = "file:" + os.path.join(ROOT_DIR_PATH, "mlruns")
+TEST_EXPERIMENT_NAME = "Pytest"
+TEST_RUN_NAME = "DecisionTree"
+
+# mlrun tags
+X_COL = "X_col"
+Y_COL = "y_col"
+RUN_ID = "run_id"
+iris_X_COL = [
```

Comment:
hahahaha not related to our project

Reply:
yea haha that's for testing let me delete it